index 22c0d86..66b2daf 100644
--- a/lua/lspsaga/provider.lua
+++ b/lua/lspsaga/provider.lua
@@ -322,7 +322,7 @@ function Finder:auto_open_preview()
       opts.row = finder_win_opts.row + height + 2
       opts.col = finder_win_opts.col
       opts.width = min_width
-      opts.height = 8
+      opts.height = 40
       if self.WIN_HEIGHT - height - opts.row < 4 then
         return
       end

diff --git a/lua/lspsaga/window.lua b/lua/lspsaga/window.lua
index 017cb00..e5d733c 100644
--- a/lua/lspsaga/window.lua
+++ b/lua/lspsaga/window.lua
@@ -54,7 +54,7 @@ local function make_floating_popup_options(width, height, opts)
   local new_option = {}

   new_option.style = 'minimal'
-  new_option.width = width
+  new_option.width = 120
   new_option.height = height

   if opts.relative ~= nil then
